stdlib/universe/universe.flux

```diff
@@ -81,7 +82,7 @@ builtin inf
 builtin length // length function for arrays
 builtin linearBins
 builtin logarithmicBins
-builtin sleep // sleep is the identity function with the side effect of delaying execution by a specified duration.
+builtin sleep // sleep is the identity function with the side effect of delaying execution by a specified duration
```

Comment:
Why is this here? A bad rebase or something else?

Reply:
I deleted the period

Reply:
This is  mistery. In `master` is there. I  think that if  you rebase that would go away. Anyways this shouldn't be a problem.

Reply:
Oooooh now I got it. The period `.` 😅 

I haven't noticed that. I am using the split view in Github and it's harder to catch that.